pkgs/applications/graphics/ktikz/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, gettext, poppler_qt5, qt5 , pkgconfig }:
+{ stdenv, fetchFromGitHub, gettext, poppler, qt5 , pkgconfig }:
```

Comment:
`poppler` is not the same as `poppler_qt5`.

Reply:
It shold be the qt5 poppler if you do libsForQt5.callPackage. I forgot to do it here though.

Reply:
But `poppler` artribute should be the glib variant, not the qt one.